components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -166,11 +179,11 @@ function loadOperationsLog(update) {
         function getLogStatusIcon(entry) {
             switch (entry) {
                 case 'COMPLETED':
-                    return 'fw-success'
-                    break;
+                    return 'fw-success';
                 case 'PENDING':
-                    return 'fw-pending'
-                    break;
+                    return 'fw-pending';
+                case 'ERROR':
+                    return 'fw-error';
                 default:
                     return 'fw-info'
```

Comment:
Add semicolon by terminating the statement

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/950